classes/artifacts/market_war_exploded/jsp/Cart.jsp

```diff
@@ -0,0 +1,22 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Anatoly
+  Date: 14.12.2017
+  Time: 15:43
+  To change this template use File | Settings | File Templates.
+--%>
```

Comment:
Please never use standart IDEA templates. Nobody cares which IDE you use. 
What should I do with this information? "To change this template use File | Settings | File Templates."